### DISCUSSION TITLE
table support

### DISCUSSION THREAD

**Discussion by amitu** [fastn-stack/fastn#477] [Ideas & RFCs]
We will create new data types: `ftd.table`, `ftd.table-row` and `ftd.table-cell`, and kernel functions, `ftd.render-table`, `ftd.render-table-row`, `ftd.render-table-cell`.

The render method will accept module `ftd.render-table-helpers` which will define methods like `render-string`, `render-header-row` etc, which can be used by end users to overrider how individual cells or rows are to look.

**Comment by amitu** [comment]
https://posit-dev.github.io/great-tables/blog/design-philosophy/

**Comment by siddhantk232** [comment]
Readers should also see https://github.com/orgs/fastn-stack/discussions/1839
